src/node-lib/curriculum-api-2023/queries/lessonDownloads/rawSyntheticUVLesson.schema.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+import { syntheticUnitvariantLessonsSchema } from "@oaknational/oak-curriculum-schema";
+
+export const rawSyntheticUVLessonSchema =
```

Comment:
omitting the null_unitvariant_id here. It would be nice is this was all done in one place but where ?